x-pack/plugins/apm/common/es_fields/apm.ts

```diff
@@ -33,6 +33,7 @@ export const SERVICE_RUNTIME_VERSION = 'service.runtime.version';
 export const SERVICE_NODE_NAME = 'service.node.name';
 export const SERVICE_VERSION = 'service.version';
 export const SERVICE_TARGET_TYPE = 'service.target.type';
+export const SERVICE_OVERFLOW_COUNT = 'service.aggregation.overflow_count';
```

Comment:
This was recently renamed to `service_transaction.aggregation.overflow_count` (https://github.com/elastic/apm-server/pull/10095):
```suggestion
export const SERVICE_OVERFLOW_COUNT = 'service_transaction.aggregation.overflow_count';
```